packages/flame/lib/src/components/position_component.dart

```diff
@@ -329,9 +329,15 @@ class PositionComponent extends Component {
 
   @override
   void preRender(Canvas canvas) {
+    canvas.save();
     canvas.transform(transformMatrix.storage);
   }
 
+  @override
+  void postRender(Canvas canvas) {
```

Comment:
This and `preRender` should have `@mustCallSuper` now

Reply:
done